server/rooms.ts

```diff
@@ -1159,10 +1159,10 @@ export class BasicChatRoom extends BasicRoom {
 		this.log.modlog(message);
 		return this;
 	}
-	hideText(userids: ID[]) {
+	hideText(userids: ID[], lineCount = 0) {
 		const cleared = this.log.clearText(userids);
```

Comment:
This should probably also be modified.

Reply:
In what way? To log the amount of lines cleared?

Reply:
Sorry, I meant in the Roomlog#clearText function (to only remove the first n lines from the scrollback log)

Reply:
Aha, I see what the function does now. Will be working on that, thanks for the heads up.